README.md

```diff
@@ -85,7 +85,7 @@ To use the Ansible OneView modules, connection properties for accessing the OneV
     "authLoginDomain": "",
     "password": "secret123"
   },
-  "api_version": 2000
+  "api_version": 2200
 }
 ```
 
```

Comment:
In line 92, it was mentioned that default api version is 600. Please modify that to

```The api_version specifies the version of the REST API to be invoked. When api_version is not specified, it will use 2200 as the default value.
```

Reply:
done